src/aviary/tools/base.py

```diff
@@ -123,9 +123,7 @@ def __str__(self) -> str:
 
 
 class ToolRequestMessage(Message):
-    role: Literal["assistant"] = Field(
-        default="assistant", description="Matching LiteLLM structure."
-    )
+    role: str = Field(default="assistant", description="Matching LiteLLM structure.")
```

Comment:
Can you type ignore this and revert to `Literal`

Let's push this change to later when we know more